src/inmanta/server/protocol.py

```diff
@@ -381,6 +384,7 @@ def start(self) -> None:
         """
             Start the server slice.
         """
+        yield super().start()
```

Comment:
The prestart(), start() and stop() function of SessionManager have return type "None". Shouldn't it be NoneGen?